src/scenes/home/termsOfService/termsOfService.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import Section from 'shared/components/section/section';
+
+class Terms extends Component {
```

Comment:
In our ongoing effort to cut weight, could you convert this to a pure function? Please see screenshot for a pattern to follow 
<img width="749" alt="screen shot 2017-10-04 at 9 14 15 am" src="https://user-images.githubusercontent.com/15932239/31177598-82cd4026-a8e4-11e7-8c8c-b7c1e24df68b.png">

Reply:
I know this was discussed on Slack, but I couldn't keep up; would you still like me to make this change?

Reply:
Yes, please do.